doc/contributing/coding-conventions.xml

```diff
@@ -620,6 +620,16 @@ args.stdenv.mkDerivation (args // {
          </para>
         </listitem>
        </varlistentry>
+       <varlistentry>
+        <term>
+         If it’s an <emphasis>icon pack</emphasis>:
```

Comment:
The official terminology is [icon theme](https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html).